README.md

```diff
@@ -906,16 +899,20 @@ associated TLS value.
 You can verify the changes by the following command-line.
 
 For firewall,
-```
+
+```bash
 firewall-cmd --list-port
 ```
 
 For SELinux,
-```
+
+```bash
 semanage port --list | grep "syslog"
 ```
+
 The newly specified port will be added to this default set.
-```
+
+```bash
```

Comment:
would be better to not have a syntax here - not sure why the processor does not allow code blocks without a syntax